src/main/java/mate/jdbc/model/Car.java

```diff
@@ -0,0 +1,74 @@
+package mate.jdbc.model;
+
+import java.util.List;
+
+public class Car {
+    private Long id;
+    private String model;
+    private Manufacturer manufacturer;
+    private List<Driver> driverList;
+
+    public Car(Long id, String model, Manufacturer manufacturer, List<Driver> driverList) {
+        this.id = id;
+        this.model = model;
+        this.manufacturer = manufacturer;
+        this.driverList = driverList;
+    }
+
+    public Car(String model, Manufacturer manufacturer, List<Driver> driverList) {
+        this.model = model;
+        this.manufacturer = manufacturer;
+        this.driverList = driverList;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getModel() {
+        return model;
+    }
+
+    public void setModel(String model) {
+        this.model = model;
+    }
+
+    public Manufacturer getManufacturer() {
+        return manufacturer;
+    }
+
+    public void setManufacturer(Manufacturer manufacturer) {
+        this.manufacturer = manufacturer;
+    }
+
+    public List<Driver> getDriverList() {
+        return driverList;
+    }
+
+    public void setDriverList(List<Driver> driverList) {
+        this.driverList = driverList;
+    }
+
+    public void setDriver(List<Driver> drivers) {
+        this.driverList = drivers;
+    }
+
+    @Override
+    public int hashCode() {
+        return super.hashCode();
+    }
```

Comment:
Your implementation of the `hashCode()` method does not seem to provide any functionality, as it simply returns the hash code of the superclass. 
Please, add custom logic to this method. The same for `equals` and `toString`